### DISCUSSION TITLE
Show unavailable dropdown actions as subdued

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#5915] [Feature Requests]
Right now, some of the dropdowns in the App omit unavailable options instead of showing them disabled (subdued). For instance: Settings > Data Model > Fields Context Menu > Duplicate Field (eg: doesn't show on File types). It would be clearer to _show_ the unavailable options (so you know they're there) and disable them... perhaps with a tooltip on why they aren't available.

**Comment by azrikahar** [comment]
Seems like #6553 (indirectly) implemented this!

It used to be hidden with the use of `v-if`:

https://github.com/directus/directus/blob/1b319706b34a6237a87a15d717a7f6cad605529b/app/src/modules/settings/routes/data-model/fields/components/field-select.vue#L102-L107

whereas that PR updated it to used disabled state:

https://github.com/directus/directus/blob/3d27ea7f50d23cb67cff7cec0c5a71dea2e318ab/app/src/modules/settings/routes/data-model/fields/components/field-select-menu.vue#L15-L20

As seen in the latest version `10.0.0`:

![image](https://github.com/directus/directus/assets/42867097/4f11c226-78b6-44ce-9784-20b729300220)

Though understandably the "tooltip on why they aren't available" didn't make the cut at that time.

Proceeding to mark this as resolved.
